### DISCUSSION TITLE
How to desactivate and remove Room Assistant on RPI ?

### DISCUSSION THREAD

**Discussion by Jens-Wymeersch** [mKeRix/room-assistant#1306] [General]
Hello,

I'm looking to move my instance on the RPI to HA but other commands are running on the PI. Can someone help me with how to remove RA from it ?

**Comment by julianlawson** [comment]
Hi, I tried the following which is effectively just a reverse of the installation instructions. This doesn't remove all of the related packages such as hcitool as you may need these for other purposes.

`
# Remove service
sudo systemctl stop room-assistant.service
sudo systemctl disable room-assistant.service
sudo rm /etc/systemd/system/room-assistant.service
sudo systemctl daemon-reload

# Remove nodes
sudo npm uninstall --global --unsafe-perm room-assistant

# Remove config files (change to -r to delete without warning)
rm -ri ~/room-assistant

`
